packages/shopify-api/adapters/__e2etests__/test_suite.ts

```diff
@@ -228,17 +240,6 @@ export const testSuite = [
       }),
     },
   },
-  {
```

Comment:
This fails with a different error in the standalone client, and is tested there.